.github/workflows/deploy-windows.yml

```diff
@@ -0,0 +1,110 @@
+name: deploy-windows
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+  release:
+    types:
+      - prereleased
+
+jobs:
+  create-binaries:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install nasm
+      run: |
+        $NASM_VERSION="nasm-2.14.02"
+        curl -LO "https://people.xiph.org/~tdaede/$NASM_VERSION-win64.zip"
+        7z e -y "$NASM_VERSION-win64.zip" -o"C:\nasm"
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable-x86_64-pc-windows-gnu
+        override: true
+    - name: Install cargo-c
+      run: |
+        cargo install cargo-c
+    - name: Set environment variables
+      run: |
+        echo "::add-path::C:\nasm"
+    - name: Build rav1e
+      run: |
+        cargo build --release
+    - name: Run cargo-c
+      run: |
+        cargo cinstall --release --destdir="C:\"
+    - name: Create zip
+      run: |
+        $RAV1E_PATH="$Env:GITHUB_WORKSPACE\target\release"
+        7z a rav1e.zip "$RAV1E_PATH\rav1e.exe" "C:\usr\local"
+    - name: Upload binaries
+      uses: actions/upload-artifact@v1
+      with:
+        name: rav1e-binaries
+        path: rav1e.zip
+
+  deploy:
+
+    needs: create-binaries
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Download the zip
+      uses: actions/download-artifact@v1
+      with:
+        name: rav1e-binaries
+    - name: Unzip rav1e Windows binaries
+      run: |
+        unzip rav1e-binaries/rav1e.zip -d rav1e-binaries
+    - name: Handle release data and binaries
+      if: github.event_name == 'push'
+      id: data
+      run: |
+        VERSION=$(head -n 1 CHANGELOG.md | tr -d "## Version ")
```

Comment:
Probably could be a good idea to take the version from the `git tag` or the `Cargo.toml`.